kodi_addon_checker/check_addon.py

```diff
@@ -399,16 +398,29 @@ def get_addon_name(xml_path):
     return (tree.get("id"), tree.get("version"))
 
 
-def _check_for_existing_addon(report: Report, addon_xml, repo_addons, branch_name):
+def _check_for_existing_addon(report: Report, addon_path, addon_branch):
     """Check if addon submitted already exists or not"""
-    addon_xml = os.path.join(addon_xml, "addon.xml")
+    addon_xml = os.path.join(addon_path, "addon.xml")
     addon_name, addon_version = get_addon_name(addon_xml)
 
-    if addon_name not in repo_addons:
-        report.add(Record(INFORMATION, "This is a new addon"))
-    else:
-        report.add(Record(INFORMATION, "%s addon already exist in branch %s with version %s."
-                          % (addon_name, branch_name, repo_addons[addon_name][0])))
+    branches = ['gotham', 'helix', 'isengard', 'jarvis', 'krypton', 'leia']
+
+    for branch in branches:
+
+        if branch >= addon_branch:
```

Comment:
I don't think we need this. We always want to see the else branch. Even if I commit a addon to a highter branch it's useful to know that we once had an ddon with the same name. Or we are clashing with aversion present in an old branch.

Reply:
I have fixed this.